dubbo-admin-backend/src/main/java/org/apache/dubbo/admin/config/ConfigCenter.java

```diff
@@ -103,7 +103,7 @@ GovernanceConfiguration getDynamicConfiguration() {
     Registry getRegistry() {
         Registry registry = null;
         if (registryUrl == null) {
-            if (StringUtils.isNotEmpty(registryAddress)) {
+            if (StringUtils.isBlank(registryAddress)) {
```

Comment:
Change the function ?   I think it should be `StringUtils.isNotBlank `

Reply:
From the exception info blow, I think the logic here is meant to check both `registrlUrl` and `registryAddress ` are empty.